usecases/monitoring/prometheus.go

```diff
@@ -144,6 +144,19 @@ type PrometheusMetrics struct {
 	TokenizerInitializeDuration *prometheus.HistogramVec
 	TokenCount                  *prometheus.CounterVec
 	TokenCountPerRequest        *prometheus.HistogramVec
+
+	// OpenAI
```

Comment:
Can we please move these metrics close to the components where is it used? Makes the components self contained. 

Some related discussion here.
https://weaviate-org.slack.com/archives/C076B3KDBQC/p1744035427852609